remme/tp/basic.py

```diff
@@ -37,6 +40,21 @@ def is_address(address):
         return False
 
 
+def add_event(context, event_type, attributes):
+    LOGGER.info("add_event")
+    context.add_event(
+        event_type=event_type,
+        attributes=attributes)
+
+
+def get_event_attributes(updated_state, header_signature):
+    content_dict = {"entities_changed": json.dumps([{**{'address': key,
```

Comment:
Fix the formatting. Not readable at all.

Reply:
Fixed